editor/editor_settings.cpp

```diff
@@ -453,6 +453,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 
 	// Theme
 	EDITOR_SETTING(Variant::STRING, PROPERTY_HINT_ENUM, "interface/theme/preset", "Default", "Default,Breeze Dark,Godot 2,Gray,Light,Solarized (Dark),Solarized (Light),Black (OLED),Custom")
+	EDITOR_SETTING(Variant::BOOL, PROPERTY_HINT_NONE, "interface/theme/enable_touchscreen_touch_area", DisplayServer::get_singleton()->is_touchscreen_available(), "")
```

Comment:
I'll check but this might make it appear on my Linux laptop with touchscreen, which would be unwanted.
We probably want to use the same criteria as for the 3D viewport touch sticks.

Reply:
The setting uses the same default as the 3D viewport touch sticks.

Reply:
Alright, it seems `DisplayServer::is_touchscreen_available()` is only implemented on Android, iOS and Web currently.

I think if/when it gets properly implemented on other platforms we might start getting complaints about those. We should probably rethink that eventually and see if we should differentiate between _supporting_ and _preferring_ touchscreen input.

```
$ rg ::is_touchscreen_available
servers/display_server.cpp
389:bool DisplayServer::is_touchscreen_available() const {
658:    ClassDB::bind_method(D_METHOD("is_touchscreen_available"), &DisplayServer::is_touchscreen_available, DEFVAL(SCREEN_OF_MAIN_WINDOW));

platform/ios/display_server_ios.mm
581:bool DisplayServerIOS::is_touchscreen_available() const {

platform/android/display_server_android.cpp
239:bool DisplayServerAndroid::is_touchscreen_available() const {

platform/web/display_server_web.cpp
591:bool DisplayServerWeb::is_touchscreen_available() const {
```